src/main/fc/cleanflight_fc.c

```diff
@@ -696,6 +696,11 @@ void taskMainPidLoop(void)
 #endif
 
 #if defined(BARO) || defined(SONAR)
+        // FIXME outdates comments?
```

Comment:
I put the comments in and they are essentially correct. They serve as a warning not to move `updateRcCommands` - doing so would cause problems with alt hold.

The whole thing is very fragile and needs refactoring though.